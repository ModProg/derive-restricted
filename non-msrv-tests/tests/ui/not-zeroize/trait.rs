#[derive(derive_where::DeriveWhere)]
#[derive_where(T)]
struct Test<T>(T);

fn main() {}
