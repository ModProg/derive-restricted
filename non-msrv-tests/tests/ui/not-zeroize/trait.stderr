error: unsupported trait, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd
 --> tests/ui/not-zeroize/trait.rs:2:16
  |
2 | #[derive_where(T)]
  |                ^
